crud/registry.go

```diff
@@ -23,11 +23,11 @@ func (r *Registry) typesMap() map[Kind]Actions {
 // An error will be returned if kind was previously registered.
 func (r *Registry) Register(kind Kind, a Actions) error {
 	if kind == "" {
-		return errors.New("kind cannot be empty")
+		return fmt.Errorf("kind cannot be empty")
 	}
 	m := r.typesMap()
 	if _, ok := m[kind]; ok {
-		return errors.New("kind '" + string(kind) + "' already registered")
+		return fmt.Errorf("kind '" + string(kind) + "' already registered")
```

Comment:
```suggestion
		return fmt.Errorf("kind %q already registered", kind)
```

This one is more serious than the other because it's patching an input parameter into the format string.